swarm/fuse/swarmfs_test.go

```diff
@@ -351,16 +344,18 @@ func (ta *testAPI) unmountWhenResourceBusy(t *testing.T) {
 
 	actualPath := filepath.Join(testMountDir, "2.txt")
```

Comment:
Sorry for late response.. Need to change the line from
   d, err := os.OpenFile(actualPath, os.O_RDWR, os.FileMode(0700))
to 
  d, err := os.OpenFile(actualPath, os.O_CREATE, os.FileMode(0700))